### PR TITLE
docs(EBS):update ebs things

### PR DESCRIPTION
**Pull request by alanthssss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,22 +4,24 @@
...
-     device_name_renamed = "/dev/xvdh"
-     mount_location = "/mnt/data"
-     jina_home = "/usr/local/jina"
-     }
    instances      = {
-     "encoder": {
-       "type": "c5.4xlarge",
-       "pip": [ "tensorflow>=2.0", "transformers>=2.6.0" ],
-       "command": "sudo apt install -y jq"
+     encoder: {
```
Comment (deepankarm) [inline README.md:8]: Changes for readme should be in https://github.com/jina-ai/terraform-jina-jinad-aws/blob/main/main.tf#L6. Please avoid changing readme directly

Comment (alanthssss) [reply README.md:8]: sure, I updated the doc automatically by
`docker run -v $PWD:/module quay.io/terraform-docs/terraform-docs:0.11.2 markdown table /module > README.md`

Comment (alanthssss) [reply README.md:8]: before this PR, I have updated https://github.com/jina-ai/terraform-jina-jinad-aws/blob/main/main.tf#L6.
but forgot to update the readme by `docker run -v $PWD:/module quay.io/terraform-docs/terraform-docs:0.11.2 markdown table /module > README.md`

**Comment by alanthssss** [comment]
I find that we need to define `availability_zone` when we use this module, I'll add it to doc now.

File: README.md (hunk 1 above)
Comment (deepankarm) [reply README.md:8]: Ah ok!

**Comment by alanthssss** [comment]
> I find that we need to define `availability_zone` when we use this module, I'll add it to doc now.

Done.

**Comment by alanthssss** [comment]
https://github.com/jina-ai/jina/issues/2082

**Pull request merged**